docs-article-templates/src/helper/module-builder.ts

```diff
@@ -82,29 +84,46 @@ export function createModuleDirectory() {
 // data used to create the module yml file.
 // check settings.json for repo value.  if there's no value, the root path directory will be considered the repo name.
 export function updateModule(units) {
-    if (learnRepoId) {
-        learnRepo = learnRepoId;
-    } else {
+    if (!learnRepoId) {
```

Comment:
Nit, and sorry if this is against current practices in here: I have a real aversion to coercing strings to bools. Would it be possible to have these more explicit, inline or some `!isEmtpy(str)` helper (homegrown, lodash, etc)?

Reply:
Good call, creating a task for this so we can capture all the string checks.